README.md

```diff
@@ -7,3 +7,178 @@ Status](https://github.com/maltezfaria/DataFlowTasks.jl/workflows/CI/badge.svg)]
+If we have a matrix A decomposed in `n x n` tiles, then the algorithm will have `n` steps. It implies that the step `i ∈ [1:n]` do `1` time (I), `(i-1)` times (II), and `(i-1)²` times (III). We illustrate the 2nd step of the algorithm in the following image.
+
+![Cholesky_Image](docs/src/Cholesky_2ndStep.png)
+
+The code of the sequential yet tiled factorization algorithm will be :
+
+```julia
+tilerange(ti, ts) = (ti-1)*ts+1:ti*ts
+function cholesky_dft!(A, ts)
+    m,n = size(A)
+    n%ts != 0 && error("Tilesize doesn't fit the matrix")
+    tn = round(Int, n/ts)
+
+    for ti in 1:tn
+        ri = tilerange(ti, ts)
+
+        # Diagonal cholesky serial factorization (I)
+        cholesky!(view(A,ri,ri))
+
+        # Left blocks update (II)
+        L = adjoint(UpperTriangular(view(A,ri,ri)))
+        for tj in ti+1:tn
+            rj = tilerange(tj, ts)
+            ldiv!(L, view(A,ri,rj))
+        end
+
+        # Submatrix update (III)
+        for tj in ti+1:tn
+            for tk in tj:tn
+                rj = tilerange(tj, ts)  ;  rk = tilerange(tk, ts)
+                mul!(view(A,rj,rk), adjoint(view(A,ri,rj)), view(A,ri,rk))
+            end
+        end
+    end
+
+    # Construct the factorized object
+    return Cholesky(A,'U',zero(LinearAlgebra.BlasInt))
+end
+```
+
+When it will come to actually parallelize the code, we would only have with DataFlowTasks to wrap function calls within a `@dspawn`, and add a synchronization point at the end. The parallelized code will be :
+
+```julia
+using DataFlowTasks
+using LinearAlgebra
+function cholesky_dft!(A, ts)
+    m,n = size(A)
+    n%ts != 0 && error("Tilesize doesn't fit the matrix")
+    tn = round(Int, n/ts)
+
+    for ti in 1:tn
+        ri = tilerange(ti)
+
+        # Diagonal cholesky serial factorization (I)
+        @dspawn cholesky!(@RW(view(A,ri,ri))) label="chol ($ti,$ti)"
+
+        # Left blocks update (II)
+        L = adjoint(UpperTriangular(view(A,ri,ri)))
+        for tj in ti+1:tn
+            rj = tilerange(tj)
+            @dspawn ldiv!(@R(L), @RW(view(A,ri,rj))) label="ldiv ($ti,$tj)"
+        end
+
+        # Submatrix update (III)
+        for tj in ti+1:tn
+            for tk in tj:tn
+                rj = tilerange(tj)  ;  rk = tilerange(tk)
+                @dspawn mul!(@RW(view(A,rj,rk)), @R(adjoint(view(A,ri,rj))), @R(view(A,ri,rk))) label="mul ($tj,$tk)"
+            end
+        end
+    end
+    DataFlowTasks.sync()
+    # Construct the factorized object
+    return Cholesky(A,'U',zero(LinearAlgebra.BlasInt))
+end
+```
+
+The code below shows how to use this `cholesky_dft!` function, how to profile the program and get the most information from the visualization. 
+
+```julia
+import DataFlowTask as DFT
+
+# DataFlowTasks environnement setup
+DFT.reset!()
+DFT.enable_log()
+DFT.setscheduler!(DFT.JuliaScheduler(50))
+
+# Context
+n  = 4096
+ts = 512
+A = rand(n, n)
+A = (A + adjoint(A))/2
+A = A + n*I
+
+# Compilation
+cholesky_dft!(copy(A), ts)
```

Comment:
We should probably add one or two lines of code here to check whether the computation is correct